.travis.yml

```diff
@@ -47,13 +50,45 @@ deploy:
   secret_access_key: "$AWS_SECRET_ACCESS_KEY_QA"
   on:
     branch: qa
+- provider: lambda
+  function_name: SierraHoldingParser-production
+  description: A function that parses holding records from the Sierra API
+  region: us-east-1
+  role: arn:aws:iam::946183545209:role/lambda-full-access
+  runtime: ruby2.7
+  timeout: 60
+  module_name: app
+  handler_name: handle_event
+  event:
```

Comment:
Some config requests:
- batch_size: 10
- maximum_record_age_in_seconds: 86400 (matches max age of message in kinesis stream)
- starting_position: $LATEST (I think the `$` is necessary)

Reply:
Oh that is weird. why the $ ?

Reply:
oh you know what, you don't. I know that it's used somewhere but I can't find it now, so you can just leave it as is